plugins/org.eclipse.sirius.common.ui/src/org/eclipse/sirius/common/ui/tools/api/util/EclipseUIUtil.java

```diff
@@ -62,7 +62,7 @@ public void run() {
             };
             if (null == Display.getCurrent()) {
                 // Getting the active workbench window through UI thread
-                workbench.getDisplay().syncExec(getActiveWorkbenchWindowRunnable);
+                workbench.getDisplay().asyncExec(getActiveWorkbenchWindowRunnable);
```

Comment:
With an async executon of the runnable, there is no garantee to have the windows in the end of the getActiveWindow.
See on the issue, we might have a better approach directly in Capella?